### DISCUSSION TITLE
Expanding deeply nested objects

### DISCUSSION THREAD

**Discussion by rbatukaev** [spatie/ray#314] [Q&A]
Is it possible to expand deeply nested objects?

![image](https://user-images.githubusercontent.com/4622415/108192522-6b213880-7125-11eb-8ea3-1b00834b6506.png)

**Comment by freekmurze** [comment]
Yes, in the latest versions you should be able to expand everything.

If it's not working for you, please create a minimal app for us that recreates the problem.
